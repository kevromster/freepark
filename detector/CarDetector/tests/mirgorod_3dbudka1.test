config = camera_nsk_mirgorod_3dbudka.txt
input = input/mirgorod_3dbudka1.png
expected = expected/mirgorod_3dbudka1.out
