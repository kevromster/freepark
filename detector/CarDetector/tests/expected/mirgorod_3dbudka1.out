[
parking id: 27
spots count: 0
]

[
parking id: 28
spots count: 0
]

[
parking id: 29
spots count: 1

{ lat: 54.853841; lon: 83.039726; complexity: NEWBIE }
]

